### ISSUE TITLE
ComputedFrom initial value seems to be missing in 0.15.0

### ISSUE THREAD

**Issue by kle-pra** [nartc/ngxtension-platform#139]
In version 0.15.0, bug fix https://github.com/nartc/ngxtension-platform/blob/main/CHANGELOG.md#bug-fixes seems to be missing, even after reinstalling 0.15.0 I only see 
this signatures exported for computedFrom:

_computed-from.d.ts:_
```ts
export declare function computedFrom<Input extends readonly unknown[], Output = Input>(sources: readonly [...ObservableSignalInputTuple<Input>], operator?: OperatorFunction<Input, Output>, injector?: Injector): Signal<Output>;
export declare function computedFrom<Input extends object, Output = Input>(sources: ObservableSignalInputTuple<Input>, operator?: OperatorFunction<Input, Output>, injector?: Injector): Signal<Output>;
```

Third argument is still of type Injector instead of  **ComputedFromOptions**.

**Comment by sysmat** [comment]
and in doc 
> By default, it needs to be called in an injection context, but it can also be called outside of it by passing the Injector in the third argument options object. 

from https://ngxtension.netlify.app/utilities/signals/computed-from/

**Comment by nartc** [comment]
Thanks @sysmat for reporting. Not sure what went wrong when `signalSlice` was there and the updated `computedFrom` was not 🫠. I've released 0.15.1 with the updated `computedFrom`.
